src/models/index.ts

```diff
@@ -238,6 +238,18 @@ export interface WeekDeliveryPredictionModel {
   days: number;
 }
 
+/**
+ * Interface for product
+ */
+export interface ProductModel { 
+  id: string | null;
+  itemGroupId: number;
+  name: string;
+  units: number;
+  unitSize: number;
+  unitName: string;
```

Comment:
Missing createdAt, updated. You can print model interfaces by turning const PRINT_MODEL_INTERFACES to true and start the application